lms/djangoapps/lti_provider/models.py

```diff
@@ -0,0 +1,14 @@
+"""
+Database models for the LTI provider feature.
+"""
+from django.db import models
+
+
+class LtiConsumer(models.Model):
```

Comment:
I'm hoping to update the diff tomorrow with the rest of your comments, but I just wanted to check on this one. I was going off the style guide at https://github.com/edx/edx-platform/wiki/Python-Guidelines - it asks for acronyms to count as one word. I don't have any strong preference either way, but I'd like to be consistent with as much of the codebase as possible. Which version is more prevalent?